quad-py/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "quad-py"
+description = "Quad python bindings"
+authors = [
+  "Alessandro Candido <candido.ale@gmail.com>",
+  "Alex Raiteri <alex.raiteri@studenti.unimi.it>",
+]
+readme = "../README.md"
+
+categories.workspace = true
+edition.workspace = true
+keywords.workspace = true
+license.workspace = true
+repository.workspace = true
+version.workspace = true
+
+[lib]
+name = "quad"
```

Comment:
```suggestion
name = "quad-py"
```

Reply:
can it have the same name as the one in quad/Cargo.toml?

Reply:
I'd like the Python package to be named `quad`, not `quad_py`.

This is not going to end up on `crates.io`, so I believe we don't have to worry about the name clash.

However, we might want to have a different name, at least for the Python package:
https://pypi.org/project/quad/
(but only eventually).